torch_xla/launch.py

```diff
@@ -0,0 +1,46 @@
+import os
+from typing import Callable, Optional, Tuple
+import warnings
+
+import torch.distributed as dist
+import torch_xla
+import torch_xla.core.xla_env_vars as xenv
+import torch_xla.distributed.xla_multiprocessing as xmp
+import torch_xla.utils.utils as xu
+import torch_xla.runtime as xr
+
+
+def spawn_wrapper(index, fn, *args, **kwargs):
+  fn(index, *args, **kwargs)
+
+
+def launch(
+    fn: Callable,
+    args: Tuple,
+    start_method: str = 'spawn',
+    debug_single_process: bool = False,
+):
+  """ Entry to launch multiprocess.
+
+  Raises:
+    NotImplementedError: SPMD is not supported yet.
+  """
+  if xr.is_spmd():
+    # TODO(piz): SPMD is specified differently from mp. Skip for now.
+    raise NotImplementedError(
+        'launch function does not support SPMD at this time')
+
+  if debug_single_process:
+    nprocs = 1
+  else:
+    nprocs = xr.global_runtime_device_count()
```

Comment:
Set this to `None` and `spawn` will do the right thing without initializing the runtime